apps/cli/src/commands/start-centralization-runtime.ts

```diff
@@ -1,23 +1,23 @@
-import Vorpal from "vorpal";
+import { Command } from 'commander';
+import inquirer from 'inquirer';
 import { dataCentralizationRuntime, sendSlackNotification } from "@sentry/core";
 
-export function startCentralizationRuntime(cli: Vorpal) {
+export function startCentralizationRuntime(cli: Command) {
     cli
-        .command('start-centralization-runtime', 'Start the data centralization runtime')
-        .action(async function (this: Vorpal.CommandInstance) {
-            const prompts: Vorpal.PromptObject[] = [
+        .command('start-centralization-runtime')
+        .description('Start the data centralization runtime')
+        .action(async () => {
+            const prompts = [
                 {
                     type: 'password',
                     name: 'mongoUri',
                     message: 'Enter the mongodb connection URI:',
                     mask: '*',
-                    optional: false
```

Comment:
We have to be sure we don't miss any previously required arguments that don't have validation.

Reply:
This has been added back in, great catch!